Mathlib/CategoryTheory/Monoidal/Internal/Types/CommGrp_.lean

```diff
@@ -0,0 +1,68 @@
+/-
+Copyright (c) 2025 Markus Himmel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Markus Himmel
+-/
+import Mathlib.CategoryTheory.Monoidal.Internal.Types.Grp_
+import Mathlib.CategoryTheory.Monoidal.CommGrp_
+
+/-!
+# `CommGrp_ (Type u) ≌ CommGrp.{u}`
+
+The category of internal commutative group objects in `Type`
+is equivalent to the category of "native" bundled commutative groups.
+
+Moreover, this equivalence is compatible with the forgetful functors to `Type`.
+-/
+
+universe v u
+
+open CategoryTheory
+
+namespace CommGrpTypeEquivalenceCommGrp
```

Comment:
It seems this would be more consistent with the current mathlib practice:
```suggestion
namespace commGrpTypeEquivalenceCommGrp
```

Reply:
I'm not sure that this would be good idea, because then the definition `inverse` in this namespace would clash with the `inverse` field of the equivalence that is defined below.

If you prefer, I can rename the namespace to `CommGrpTypeEquivalenceCommGrpAux`, but none of the other analogous constructions for monoid objects, commutative objects and group objects do this, so I would rename them all.

Reply:
Ok, let us keep it like this.